servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/Karaf2xPublishController.java

```diff
@@ -131,9 +136,11 @@ public int publishModule(int kind, int deltaKind, IModule[] modules, IProgressMo
 		
 		boolean isBinaryObject = ServerModelUtilities.isBinaryModule(modules);
 		if( !isBinaryObject ) {
-			for (IModule module : modules) {
-				KarafUtils.runBuild(GOALS, module, subMonitor.newChild(1));
-			}
+		    // commented out after the updates to m2e-wtp which now handles
```

Comment:
Here in this case I opt for removing instead of commenting. But I guess the whole if case needs a rework then.